packages/rest-typings/src/v1/channels/ChannelsCloseProps.ts

```diff
@@ -0,0 +1,28 @@
+import Ajv from 'ajv';
+
+const ajv = new Ajv();
+
+export type ChannelsCloseProps = { roomId: string } | { roomName: string };
+
+const ChannelsClosePropsSchema = {
+	oneOf: [
+		{
+			type: 'object',
+			properties: {
+				roomId: { type: 'string' },
+			},
+			required: ['roomId'],
+			additionalProperties: false,
+		},
+		{
+			type: 'object',
+			properties: {
+				roomName: { type: 'string' },
+			},
+			required: ['roomName'],
+			additionalProperties: false,
+		},
+	],
```

Comment:
```suggestion
	type: 'object',
	properties: {
		roomId: { type: 'string' },
		roomName: { type: 'string' },
	},
	oneOf: [{ required: ['roomId'] }, { required: ['roomName'] }],
	additionalProperties: false,
```
I believe it's easier to read like this.